src/guide/modules.md

```diff
@@ -100,32 +100,32 @@ from "list" include List as Stdlist
 Stdlist.length([1, 2, 3]) // 3
 ```
 
-### Including All Values
+### Bring Values from a Module into Scope
 
-If we want to bring all of the values from another module into scope in our program, we can use an asterisk.
+Individual items in a module can be brought into scope by using the `use` keyword, placing the desired items in curly braces. Similar to with the `provide { ... }` syntax, we need to prefix types with `type` and modules with `module`
```

Comment:
I think this reads wierd an `and` might help, along with getting rid of `using the use`.
```suggestion
Individual items in a module can be brought into scope with the `use` keyword, and placing the desired items in curly braces. Similar to with the `provide { ... }` syntax, we need to prefix types with `type` and modules with `module`
```